order.php

```diff
@@ -33,67 +25,52 @@
+              <div class="bnomics-address">
+                    <div class="bnomics-address-text" ng-hide="address_copyshow" ng-cloak><?=__('To this bitcoin{{crypto.name | lowercase}} address', 'edd-blockonomics')?></div>
+                    <div class="bnomics-copy-address-text" ng-show="address_copyshow" ng-cloak><?=__('Copied to clipboard', 'edd-blockonomics')?></div>
+                    <ul ng-click="blockonomics_address_click()" id="bnomics-address-input" class="bnomics-address-input">
+                          <li id="bnomics-address-copy">{{order.address}}</li>
+                    </ul>
+                  </div>
             <!-- Countdown Timer -->
                 <div ng-cloak ng-hide="order.status != -1" class="bnomics-progress-bar-wrapper">
                   <div class="bnomics-progress-bar-container">
                     <div class="bnomics-progress-bar" style="width: {{progress}}%;"></div>
                   </div>
                 </div>
-                <span class="ng-cloak bnomics-time-left" ng-hide="order.status != -1">{{clock*1000 | date:'mm:ss' : 'UTC'}} min left to pay your order</span>
+                <span class="ng-cloak bnomics-time-left" ng-hide="order.status != -1">{{clock*1000 | date:'mm:ss' : 'UTC'}} min</span>
               </div>
         <!-- Blockonomics Credit -->
-            <div class="bnomics-powered-by">
-              <?=__('Powered by ', 'edd-blockonomics')?>Blockonomics
-            </div>
-              </div>
-            </div>
-          </div>
-
-          <?php if (edd_get_option('edd_blockonomics_altcoins', 0)) : ?>
-          <div class="bnomics-altcoin-pane" ng-style="{'border-left': (altcoin_waiting)?'none':''}" ng-hide="show_altcoin != 1">
-            <div class="bnomics-altcoin-bg">
-                <div class="bnomics-altcoin-bg-color" ng-hide="altcoin_waiting" ng-cloak>
-                 <div class="bnomics-altcoin-info-wrapper">
-                  <span class="bnomics-altcoin-info" ><?=__('Select your preferred <strong>Altcoin</strong> then click on the button below.', 'edd-blockonomics')?></span>
-                 </div>
-                 </br>
-                 <!-- Coin Select -->
-                 <div class="bnomics-address">
-                   <select ng-model="altcoinselect" ng-options="x for (x, y) in altcoins" ng-init="altcoinselect='Ethereum'"></select>
-                 </div>
-                 <div class="bnomics-altcoin-button-wrapper">
-                  <a ng-click="pay_altcoins()" href=""><button><i class="cf" ng-hide="altcoinselect!='Ethereum'" ng-class={'cf-eth':'{{altcoinselect}}'!=''} ></i><i class="cf" ng-hide="altcoinselect!='Litecoin'" ng-class={'cf-ltc':'{{altcoinselect}}'!=''} ></i> <?=__('Pay with', 'edd-blockonomics')?> {{altcoinselect}}</button></a>
-                 </div>
+                <div class="bnomics-powered-by">
```

Comment:
The Powered by Blockonomics section can be moved outside of `bnomics-btc-info` in order to remain centered on the page.
https://github.com/blockonomics/woocommerce-plugin/blob/master/templates/blockonomics_checkout.php#L99-L103